docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -71,8 +89,24 @@ Follow these steps to configure and enable SAML integration:
 
    The SAML standard recommends using a digital signature for some types of messages, like authentication or logout requests to avoid [man-in-the-middle attacks](https://en.wikipedia.org/wiki/Man-in-the-middle_attack).
 
-1. Click **Next: Connect Grafana with Identity Provider** and complete the section.
+1. Click **Next: Connect Grafana with Identity Provider**.
+
+### 3. Connect Grafana with Identity Provider Section
+
+1. Configure IdP using Grafana Metadata
+   1. Copy the **Metadata URL** and provide it to your SAML IdP to establish a connection between Grafana and the IdP.
+      - The metadata URL contains all the necessary information for the IdP to establish a connection with Grafana.
+   1. Copy the **Assertion Consumer Service URL** and provide it to your SAML IdP.
+      - The Assertion Consumer Service URL is the endpoint where the IdP sends the SAML assertion after the user has been authenticated.
+   1. If you want to use the **Single Logout** feature, copy the **Single Logout Service URL** and provide it to your SAML IdP.
+1. Finish configuring Grafana using IdP data
+1. Provide Grafana metadata to your SAML IdP.
+   - The metadata contains all the necessary information for the IdP to establish a connection with Grafana.
+   - This can be provided as Base64-encoded value, a path to a file, or as a URL.
```

Comment:
Question: I'm slightly confused about these two. Isn't the first point already about configuring the IdP with Grafana metadata? Or does it not cover everything? And which bits need to be configured in Grafana here?

Reply:
You are right that it might seem a bit confusing, but that's the name of the section on the SAML screen.

I added some indentation to make this clear. I also changed the direction of the metadata provisioning. In this step 2 `Finish configuring Grafana using IdP data` we're providing the idp metadata into Grafana.